recipe/bld.bat

```diff
@@ -7,7 +12,10 @@ cmake .. ^
   -D CMAKE_PREFIX_PATH=%LIBRARY_PREFIX% ^
   -DASSIMP_BUILD_ASSIMP_TOOLS=OFF ^
   -DASSIMP_BUILD_TESTS=OFF ^
```

Comment:
```suggestion
  -DASSIMP_BUILD_TESTS=OFF ^
  -DASSIMP_BUILD_ZLIB=OFF ^
```
The windows build builds its own zlib by default.
https://github.com/assimp/assimp/blob/v5.4.3/CMakeLists.txt#L148-L158